lib/gui/os/dialog/services/dialog.js

```diff
@@ -113,20 +113,28 @@ module.exports = function($q, SupportedFormatsModel) {
    * });
    */
   this.showWarning = (options) => {
+    const BUTTONS = [
+      options.confirmationLabel,
+      options.rejectionLabel
+    ];
+
+    const BUTTON_CONFIRMATION_INDEX = _.indexOf(BUTTONS, options.confirmationLabel);
+    const BUTTON_REJECTION_INDEX = _.indexOf(BUTTONS, options.rejectionLabel);
+
     return $q((resolve) => {
       electron.remote.dialog.showMessageBox(currentWindow, {
         type: 'warning',
         buttons: [
```

Comment:
Given you've constructed the array above, could this just be `buttons: BUTTONS` ?